dvc/commands/experiments/apply.py

```diff
@@ -4,21 +4,13 @@
 from dvc.cli import completion
 from dvc.cli.command import CmdBase
 from dvc.cli.utils import append_doc_link
-from dvc.ui import ui
 
 logger = logging.getLogger(__name__)
 
 
 class CmdExperimentsApply(CmdBase):
     def run(self):
-        if not self.args.force:
-            ui.write(
-                "The --no-force option is deprecated and will be removed in a future"
-                " DVC release. To revert the result of 'exp apply', run:\n"
-                "\n\tgit reset --hard\n"
-                "\tgit stash apply refs/exps/apply/stash\n"
-            )
-        self.repo.experiments.apply(self.args.experiment)
+        self.repo.experiments.apply(self.args.experiment, force=self.args.force)
```

Comment:
Setting `force` here does not anything, the underlying `experiments.apply` behavior was changed to always do the force apply. Since the DVC `checkout` now needs force to allow removing untracked files, that flag should just always be set internally by `experiments.apply`.

Basically, the intent is to not have a CLI level option for `exp apply --force/--no-force` at all